src/main/webapp/resources/js/apis/projects/projects.js

```diff
@@ -68,20 +40,3 @@ export async function updateProjectAttribute({ projectId, field, value }) {
 export async function getProjectInfo(projectId) {
```

Comment:
Isn't this getting the same data as this code from `project.js`:

  `getProjectDetails: build.query({
    query: (projectId) => ({
      url: "",
      params: { projectId },
    }),
    providesTags: ["Project"],
  }),`
     
The only place using the `getProjectInfo` here is in ReferenceFile.jsx. I have not looked over the PR for that yet but if you have already fixed this there then it's alright

Reply:
Yeah, just wait for that PR :)  You have a very good eye for details

Reply:
Sounds good. Thanks :)